### DISCUSSION TITLE
Python Scripts Open Into Word Blocks

### DISCUSSION THREAD

**Discussion by tomt453** [pybricks/support#1854] [Q&A]
Hi Pybricks Community! I've noticed that any python script files open into word blocks when imported into pybricks. These scripts were originally created in Pybricks and then exported. Is there a setting or step that I'm missing? I'm working around the issue by opening the files in Notepad++ and then copy/pasting into a new file in Pybricks. Thanks!

**Comment by laurensvalk** [comment]
Can you share one example file where this occurs? Or perhaps a screen recording of this happening? When I try this, the program successfully open as Python scripts. I'd love to reproduce this so we could fix any possible issue.
